src/components/SVGViewer/stories/SVGViewer.stories.tsx

```diff
@@ -4,7 +4,7 @@ import { storiesOf } from '@storybook/react';
 import React from 'react';
 import styled from 'styled-components';
 import { ZoomCenter } from '../../../interfaces';
-import { SVG } from '../../../mocks/svg-viewer';
+import { CUSTOM_SVG_FILE, SVG } from '../../../mocks/svg-viewer';
```

Comment:
What is the difference between `CUSTOM_SVG_FILE` and `SVG`?

Reply:
Its just file with a bunch of nodes/lines removed.
(it . used to be the cognite icon, since tbh any svg works)